multi_import/formats.py

```diff
@@ -130,6 +130,15 @@ def pre_read(self, file_object):
         file_object = strings.normalize_string(file_object)
         return file_object
 
+    def detect(self, file_handler, file_contents):
+        file_object = self.get_file_object(file_handler, file_contents)
+        try:
+            Dataset().load(file_object, "csv")
+            return True
```

Comment:
Should you return `super().detect(file_handler, file_contents)` here so that the normal checks are still being done?

Reply:
In this particular case it is fine to not return a super because the intent really is to override the default implementation. The original detect call uses a [CSV sniffer](https://github.com/jazzband/tablib/blob/master/src/tablib/formats/_csv.py#L58) to try and figure out whether the file is a CSV file or not. I think the intent behind using the sniffer is to detect the file format without having to read the entire file - [which we do anyways](https://github.com/sdelements/django-multi-import/blob/master/multi_import/helpers/files.py#L29C32-L29C32). Because the sniffer has false positives, we are instead opting to just attempt to parse the file as a CSV and if errors pop up we know it is not a CSV

Reply:
I'm glad we're all coming out of retirement for this MR.

Reply:
Thanks @funkaoshi for looking at this. I ran a local debugger to check how self and super are working here. Turns out self.detect calls the method in the parent class (TabLibFileFormat) since the method is non-existent on the child class, it looks for the method in the parent class. Super on the other hand requires that we call it with some args and then call the get_file_object method.

```
super_return = super(CsvFormat, self).detect(file_handler, file_contents)
self_return = self.detect(file_handler, file_contents)
```
I ran a check for ```super_return == self_return``` and they are equal. In case I am missing other contextual factors and possibly my super call understanding is flawed, then we can use the super instead and I would like to be enlightened how super really works.